### PR TITLE
usefull button

### PR DESCRIPTION
**Pull request by kardagan**
- adding some minimize and maximize button next to the close button

- adding the pdo connection name, and build filter on them

### CODE REVIEW

**Comment by kardagan** [comment]
i did have change the postion of the button like you've ask.
but i didn't find hover on the other button.
for the base64 encoded image tel me if you want t obligatory or not.

**Comment by barryvdh** [comment]
Base64 is needed indeed :)

File: src/DebugBar/Resources/debugbar.js [hunk 1]
```diff
@@ -481,14 +481,27 @@ if (typeof(PhpDebugBar) == 'undefined') {
...
+                self.minimize();
+            });
+
+            // maximize button
+            this.$maximizebtn = $('<a href="javascript:" />').addClass(csscls('maximize-btn') ).appendTo(this.$headerRight);
+            this.$maximizebtn.click(function() {
+                self.restore();
+            });
+
+
```
Comment (GrahamCampbell) [inline src/DebugBar/Resources/debugbar.js]: there should only be 1 blank line here, not two

**Comment by kardagan** [comment]
i've made the change

File: src/DebugBar/Resources/debugbar.css [hunk 2]
```diff
@@ -65,7 +65,6 @@ div.phpdebugbar-resize-handle {
 div.phpdebugbar-closed, div.phpdebugbar-minimized{
   border-top: 1px solid #ccc;
 }
-
```
Comment (GrahamCampbell) [inline src/DebugBar/Resources/debugbar.css]: ?

**Comment by barryvdh** [comment]
Thanks, merged!

**Pull request merged**